src/components/HomePage/Banner.js

```diff
@@ -151,6 +153,23 @@ function renderInfo(classes) {
 	);
 }
 
+const Image = () => {
+	const data = useStaticQuery(graphql`
+    query {
+      placeholderImage: file(relativePath: { eq: "powellBackground.png" }) {
+        childImageSharp {
+          fluid(maxWidth: 300) {
+            ...GatsbyImageSharpFluid
+          }
+        }
+      }
+    }
+  `);
+
+	return <Img fluid={data.placeholderImage.childImageSharp.fluid} />;
```

Comment:
Rather than returning `<Img>` here we'll just inline this in the `Banner` component. See the suggestion below